MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -242,7 +245,9 @@ internal bool HandlePacket(int packetID, List<byte> packetData)
                         float pitch = dataTypes.ReadNextFloat(packetData);
                         byte locMask = dataTypes.ReadNextByte(packetData);
 
-                        if (handler.GetTerrainEnabled())
+                        // player pos is needed for calculate entity distance
+                        // modified by reinforce
+                        if (handler.GetTerrainEnabled() || true)
```

Comment:
When terrain is disabled, entity handling should be disabled as well, thus not requiring this condition bypass. Or if entity handling could work without terrain handling (unsure if it would), add a setting for enabling entity handling and use `|| handler.GetEntitiesEnabled()` instead of `|| true` 😉